packages/docs/docs/bots/monitoring-bots.md

```diff
@@ -0,0 +1,24 @@
+# Monitoring Bots
+
+Bots allow for powerful functionality in your app, so it is very important to track what exactly each bot has done. This is possible using a bot's `AuditEvent` resources, which track each time the bot runs.
+
+## Viewing a Bot's Events
+
+To monitor your bots, navigate to the Bot resource page at https://app.medplum.com/Bot. This page will display all of the bots that are a part of your project.
+
+Choose which bot's events you would like to view and click on it, bringing you to the bot resource page.
+
+From this page, navigate to the `Event` tab. This tab will display all of the `AuditEvent` resources associated with the current bot. These events represent every time the bot has been run or triggered. This page displays the outcomes from the bot being run, including anything that is printed to the console as part of the bot's functionality.
```

Comment:
Note that you can find this at https://app.medplum.com/Bot/<id>/event

Reply:
Add a section "Audit Events Missing" - in that section note that it is possible that there are no AuditEvents stored in the database.  This is often because the Bot is configure to write only to the logs, to enable at scale bot execution.  To ensure that the AuditEvents show up in the console configure your bot AuditEventDestination to [resource](https://medplum-www-git-fork-ksmith94-monitoring-bots-medplum.vercel.app/docs/api/fhir/medplum/bot).  Related.  
 https://medplum-www-git-fork-ksmith94-monitoring-bots-medplum.vercel.app/docs/agent#preparing-for-scale

Reply:
This section has been added. Let me know if you see any issues with it or want any changes